test/watcher_test.js

```diff
@@ -3,8 +3,28 @@ import chai from 'chai';
 import sinonChai from 'sinon-chai';
 import Sinon from 'sinon';
 import Watcher from '../lib/watcher';
+import broccoli from '..';
+const Builder = broccoli.Builder;
+const multidepRequire = require('multidep')('test/multidep.json');
 import SourceNodeWrapper from '../lib/wrappers/source-node';
 
+const Plugin = multidepRequire('broccoli-plugin', '1.3.0');
+const Merge = multidepRequire('broccoli-merge-trees', '4.1.0');
+
+function defer() {
+  let deferred = {};
+  let promise = new Promise(function(resolve, reject) {
+    deferred.resolve = resolve;
+    deferred.reject = reject;
+  });
+  deferred.promise = promise;
+  return deferred;
+}
+
+function sleep() {
+  return new Promise(resolve => setTimeout(resolve, 10));
+}
```

Comment:
When I write a `sleep` function like this, I generally implement accepting a timeout value (defaulting if appropriate):

```suggestion
function sleep(timeout = 10) {
  return new Promise(resolve => setTimeout(resolve, timeout));
}
```